pkg/broker/broker.go

```diff
@@ -81,6 +83,19 @@ func NewAnsibleBroker(
 	return broker, nil
 }
 
+func (a AnsibleBroker) getServiceInstance(instanceUUID uuid.UUID) (*apb.ServiceInstance, error) {
```

Comment:
Can you leave a note here to combine GetServiceInstance and getServiceInstance later?